clients/venice-push-job/src/main/java/com/linkedin/venice/vpj/VenicePushJobConstants.java

```diff
@@ -324,6 +324,14 @@ private VenicePushJobConstants() {
    */
   public static final String TARGETED_REGION_PUSH_LIST = "targeted.region.push.list";
 
+  /**
+   * Config to enable target region push with deferred version swap
+   * In this mode, the VPJ will push data to all regions and only switch to the new version in a single region.
+   * The single region is decided by the store config in {@link StoreInfo#getNativeReplicationSourceFabric()}}.
```

Comment:
We should have an override for this, either we can reuse the `"targeted.region.push.list"` or create a new one.

Reply:
Yeah agreed, we're reusing `targeted.region.push.list` to override which regions we will switch to the new version on. Updated the comment to reflect that it can be overridden with `targeted.region.push.list`